megatron/megatron/core/parallel_state.py

```diff
@@ -1847,7 +1855,7 @@ def get_data_modulo_expert_parallel_rank(with_context_parallel=False):
     """Return my rank for the context parallel group."""
     para_ctx = get_parallel_context() 
     if para_ctx is not None:
-        return para_ctx.get_data_modulo_expert_parallel_rank(with_context_parallel)
```

Comment:
Why does this code remove the `with_context_parallel` argument?

Reply:
The `get_data_modulo_expert_parallel_rank` of `ParallelContext` doesn't support the argument  `with_context_parallel`.   Removing this argument is a temporary solution. The newest commit has added the `dp_modulo_exp` group.